src/flamegraphs.jl

```diff
@@ -82,26 +80,17 @@ function pprof(fg::Node{NodeData},
     funcs = Dict{UInt64, Function}()
 
     seen_locs = Set{UInt64}()
-    locs  = Dict{UInt64, Location}()
-    locs_from_c  = Dict{UInt64, Bool}()
+    locs = Dict{UInt64, Location}()
+    locs_from_c = Dict{UInt64, Bool}()
 
     sample_type = [
         ValueType!("events", "count"), # Mandatory
     ]
 
-    prof = PProfile(
-        sample = [], location = [], _function = [],
-        mapping = [], string_table = [],
-        sample_type = sample_type, default_sample_type = 1, # events
-        period = period, period_type = ValueType!("cpu", "nanoseconds")
-    )
-
-    if drop_frames !== nothing
-        prof.drop_frames = enter!(drop_frames)
-    end
-    if keep_frames !== nothing
-        prof.keep_frames = enter!(keep_frames)
-    end
+    period_type = ValueType!("cpu", "nanoseconds")
+    drop_frames = isnothing(drop_frames) ? 0 : enter!(drop_frames)
+    keep_frames = isnothing(keep_frames) ? 0 : enter!(keep_frames)
```

Comment:
Is 0 the correct default value for these when there is no provided value? 🤔

Reply:
Yeah, so when default value is not changed in the proto file via options, 0 is the default for numeric scalars